docs/getting-started/first-source.md

```diff
@@ -18,26 +18,50 @@ Create the CloudEvents Player Service:
     --image ruromero/cloudevents-player:latest \
     --env BROKER_URL=http://broker-ingress.knative-eventing.svc.cluster.local/default/example-broker
     ```
+    ==**Expected Output**==
+    ```{ .bash .no-copy }
+    Service 'cloudevents-player' created to latest revision 'cloudevents-player-vwybw-1' is available at URL:
+    http://cloudevents-player.default.127.0.0.1.nip.io
+    ```
+
+    ??? question "Wait, my Revision is named something different!"
+        Since we didn't assign a `revision-name`, Knative Serving automatically created one for us, it's ok if your Revision is named something different.
 
 === "YAML"
 
     ```bash
-    //TODO
+    apiVersion: serving.knative.dev/v1
+    kind: Service
+    metadata:
+      name: cloudevents-player
+    spec:
+      template:
+        metadata:
+          annotations:
+            autoscaling.knative.dev/minScale: "1"
+        spec:
+          containers:
+            - image: ruromero/cloudevents-player:latest
+              env:
+                - name: BROKER_URL
+                  value: http://broker-ingress.knative-eventing.svc.cluster.local/default/example-broker
     ```
 
-==**Expected Output**==
-```{ .bash .no-copy }
-Service 'cloudevents-player' created to latest revision 'cloudevents-player-vwybw-1' is available at URL:
-http://cloudevents-player.default.127.0.0.1.nip.io
-```
-??? question "Wait, my `Revision` is named something different!"
-    Since we didn't assign a `revision-name`, Knative Serving automatically created one for us, it's ok if your `Revision` is named something different.
+    Once you've created your YAML file (named something like "cloudevents-player.yaml"):
+    ``` bash
+    kubectl apply -f cloudevents-player.yaml
+    ```
+
+    ==**Expected Output**==
+    ```{ .bash .no-copy }
+    service.serving.knative.dev/cloudevents-player created
+    ```
 
 ## Examining the CloudEvents Player
-We can use the CloudEvents Player to send and receive CloudEvents. If you open the [Service URL](http://cloudevents-player.default.127.0.0.1.nip.io){target=_blank} in your browser, you should be greeted by a form titled "Create event."
+**We can use the CloudEvents Player to send and receive CloudEvents.** If you open the [Service URL](http://cloudevents-player.default.127.0.0.1.nip.io){target=_blank} in your browser, you should be greeted by a form titled "Create event."
```

Comment:
**You can use the CloudEvents Player to send and receive CloudEvents.** When you open the [Service URL](http://cloudevents-player.default.127.0.0.1.nip.io){target=_blank} in your browser, the **Create event** form appears.